lib/src/git.rs

```diff
@@ -136,14 +176,27 @@ pub fn import_some_refs(
             }
         };
         let id = CommitId::from_bytes(git_commit.id().as_bytes());
-        new_git_heads.insert(id.clone());
+        new_git_heads.insert(id.clone()); // TODO/unrelated: why is this before checking for git_ref_filter?
```

Comment:
Note to self: I want to leave this TODO in until I look at the function more carefully, but it should say "TODO: document why ...".